WTWSMS/common/religions/00_religions.txt

```diff
@@ -2516,12 +2558,13 @@ pagan_group = {
 		female_temple_holders = yes
 		feminist = yes
 		defensive_attrition = yes
-		max_consorts = 3
 		peace_prestige_loss = yes # Lose Prestige while at peace
 		raised_vassal_opinion_loss = no
 		allow_looting = yes
 		reformed = celtic_pagan_reformed
 		aggression = 1.5 # AI aggression factor
+		bs_marriage = yes
+		pc_marriage = yes
```

Comment:
They did practice incest.

Maelgwn Gwynedd married his cousin Gwallwyn (Gildas' De Excidio)
Geraint ap Erbin married his unnamed niece (Harleian Genealogies)
Gworthigern (Vortigern) married his daughter
Vortipor married his sister and had incestuous relations with his daughter (Gildas' De Excidio)

I don't think it's not too much of a stretch to allow pc and bs incest for everyone.